bluebutton-data-pipeline-fhir-load/src/test/java/gov/hhs/cms/bluebutton/datapipeline/fhir/transform/DataTransformerTest.java

```diff
@@ -639,40 +688,119 @@ public void transformInsertPartDEvent() throws FHIRException {
 		Assert.assertNotNull(pdeBundleWrapper.getResult());
 
 		Bundle pdeBundle = pdeBundleWrapper.getResult();
-		Assert.assertEquals(1, pdeBundle.getEntry().size());
+		Assert.assertEquals(5, pdeBundle.getEntry().size());
 		BundleEntryComponent eobEntry = pdeBundle.getEntry().stream()
 				.filter(r -> r.getResource() instanceof ExplanationOfBenefit).findAny().get();
 		Assert.assertEquals(HTTPVerb.PUT, eobEntry.getRequest().getMethod());
+
 		ExplanationOfBenefit eob = (ExplanationOfBenefit) eobEntry.getResource();
-		Assert.assertEquals("Patient/" + record.beneficiaryId, eob.getPatient().getReference());
+		Assert.assertEquals("ExplanationOfBenefit/" + pdeRecord.partDEventId, eob.getId());
+		// TODO verify eob.type once STU3 available
+		Assert.assertEquals("Patient/" + pdeRecord.beneficiaryId, eob.getPatient().getReference());
+		Assert.assertEquals(Date.valueOf(pdeRecord.paymentDate.get()), eob.getPaymentDate());
 
 		ItemsComponent rxItem = eob.getItem().stream().filter(i -> i.getSequence() == 1).findAny().get();
-		Assert.assertEquals("RXCINV", rxItem.getType().getCode());
-		Assert.assertEquals(Date.valueOf(record.prescriptionFillDate), rxItem.getServicedDateType().getValue());
-		// TODO rest of values
+		// Default case has compound code = not a compound
+		Assert.assertEquals("RXDINV", rxItem.getType().getCode());
+		Assert.assertEquals(Date.valueOf(pdeRecord.prescriptionFillDate), rxItem.getServicedDateType().getValue());
+
+		// Default case has drug coverage status code as Covered
+		assertAdjudicationEquals(DataTransformer.CODED_ADJUDICATION_PART_D_COVERED,
+				pdeRecord.partDPlanCoveredPaidAmount, rxItem.getAdjudication());
+		assertAdjudicationNotPresent(DataTransformer.CODED_ADJUDICATION_PART_D_NONCOVERED_SUPPLEMENT,
+				rxItem.getAdjudication());
+		assertAdjudicationNotPresent(DataTransformer.CODED_ADJUDICATION_PART_D_NONCOVERED_OTC,
+				rxItem.getAdjudication());
+		assertAdjudicationEquals(DataTransformer.CODED_ADJUDICATION_PATIENT_PAY, pdeRecord.patientPaidAmount,
+				rxItem.getAdjudication());
+		assertAdjudicationEquals(DataTransformer.CODED_ADJUDICATION_OTHER_TROOP_AMOUNT,
+				pdeRecord.otherTrueOutOfPocketPaidAmount, rxItem.getAdjudication());
+		assertAdjudicationEquals(DataTransformer.CODED_ADJUDICATION_LOW_INCOME_SUBSIDY_AMOUNT,
+				pdeRecord.lowIncomeSubsidyPaidAmount, rxItem.getAdjudication());
+		assertAdjudicationEquals(DataTransformer.CODED_ADJUDICATION_PATIENT_LIABILITY_REDUCED_AMOUNT,
+				pdeRecord.patientLiabilityReductionOtherPaidAmount, rxItem.getAdjudication());
+		assertAdjudicationEquals(DataTransformer.CODED_ADJUDICATION_TOTAL_COST, pdeRecord.totalPrescriptionCost,
+				rxItem.getAdjudication());
+		assertAdjudicationEquals(DataTransformer.CODED_ADJUDICATION_GAP_DISCOUNT_AMOUNT, pdeRecord.gapDiscountAmount,
```

Comment:
Glad to see you using these helper methods! Please keep an eye out for additional ones we can add that will make the tests more readable. (I'm sure my tests could still use one or two more.)